src/src/components/app-projects/app-projects.jsx

```diff
@@ -0,0 +1,152 @@
+import { Link } from 'react-router-dom';
+import appProjectService from '../../services/appProjectService';
+import { useEffect, useState } from 'react';
+import notify from '../../services/notify';
+import { Confirm } from 'semantic-ui-react';
+import enums from '../../enums/enums';
+import styles from './app-projects.module.css';
+
+const AppProjects = () => {
+  const [appProjects, setAppProjects] = useState([]);
+  const [confirmShown, setConfirmShown] = useState(false);
+  const [productIdToDelete, setProductIdToDelete] = useState(0);
+
+  const handleConfirm = () => {
+    appProjectService
+      .delete(productIdToDelete)
+      .then(() => {
+        getData();
+        notify.success('App project successfully removed');
+      })
+      .catch((err) => notify.error(`Operation failed. ${err ?? ''}`));
+    setConfirmShown(false);
+  };
+  const handleCancel = () => setConfirmShown(false);
+
+  useEffect(() => getData(), []);
+
+  const getData = () => {
+
+    // eslint-disable-next-line no-debugger
+    debugger
+    appProjectService
+      .get()
+      .then((resp) => {
+        // eslint-disable-next-line no-debugger
+        debugger
+        setAppProjects(resp.data);
+      })
+      .catch((err) => {
+        
+        // eslint-disable-next-line no-debugger
+        debugger
```

Comment:
debugger and commented line could be removed

Reply:
Done.